sotopia/envs/evaluators.py

```diff
@@ -150,6 +150,98 @@ def minus_ten_to_zero_validator(cls, v: tuple[str, int]) -> tuple[str, int]:
         return v
 
 
+class EvaluationByGroup(BaseModel):
+    effectiveness: tuple[str, int] = Field(
+        ...,
+        description="""Consider features of verbosity (e.g. word count, sentence lengths, readability)
+         to analyze the interaction episode between the agents and use this as a method in evaluating 
+         the efficiency of this episode. Higher levels of efficiency should be reflected with focused, 
+         coherent, and collaborative communication that advances the team's objectives. Lower efficiency 
+         can be marked by disjointed, repetitive, or ineffective exchanges that hinder or delay the progress
+         in attaining the goal."""
+        """Output your reasoning process to the 'reasoning' field. Output an integer score ranging from 0 and 
```

Comment:
The format here is a little bit ambiguous, please refer to the latest prompt to make things clear: https://github.com/sotopia-lab/sotopia/blob/1939bc6963f3c317a9bf62600126d24965f9969c/sotopia/envs/evaluators.py#L156